pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -285,6 +290,56 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 	return nil
 }
 
+// HasRoute returns false if there is a route to a different revision but not this revision,
```

Comment:
- this seems to return true even if there are other routes in addition (e.g. route can have a red/blue split) and returns false only if all the traffic is directed to a different revision on all routes.
- the name does not correspond to the contents
- does this method have to be public?

Reply:
I rewrote HasRoute() to IsReplaced(), which should read better.

It is public for testing.